automatic_gadget_generator/gadget.xml

```diff
@@ -71,8 +71,10 @@
             var experiments = [];
             var checkboxes = [];
             var configurationArray = 'var configurationArray = [];\n';
-            var slidersArray = 'var slidersArray = [ ';
+            var actuatorsArray = '';
+            var sensorsArray = 'var sensorsArray = [];\n';
             var initialOption;
+            var graasp = 1;
```

Comment:
add empty line here. 
And be consistent in leaving empty line or not for lines 88 and 92. I would prefer to leave an empty line on ws.open and ws.onmessage for clarity